python/compiler/aiecc/configure.py.in

```diff
@@ -16,6 +16,8 @@ aie_disable_compile = not pythonize_bool("@AIECC_COMPILE@")
 aie_unified_compile = True
 host_disable_compile = not pythonize_bool("@AIECC_HOST_COMPILE@")
 host_architecture = os.getenv("LLVM_HOST_TRIPLE", "@LLVM_HOST_TRIPLE@")
+hsa_dir = os.getenv("HSA_RUNTIME_64_DIR", "@hsa-runtime64_DIR@")
+libxaie_x86_hsa_dir = os.getenv("LIBXAIE_X86_HSA_DIR", "@LibXAIE_x86_64-hsa_DIR@")
```

Comment:
Don't need a separate libxaie_x86 path just for hsa. We build against one aie-rt and use that one.

Reply:
Decided to do this in another PR.